app/Models/Loggable.php

```diff
@@ -121,8 +129,17 @@ public function logCheckin($target, $note)
             $log->item_type = License::class;
             $log->item_id = $this->license_id;
         } else {
+
             $log->item_type = static::class;
             $log->item_id = $this->id;
+
+            if (static::class == Asset::class) {
+                if ($asset = Asset::find($log->item_id)) {
+                    \Log::debug('Increment the checkin count for asset: '.$log->item_id);
+                    $asset->increment('checkin_counter', 1);
+                }
+            }
+
```

Comment:
I'd say the same thing here, but I don't know if we ever finished extracting a checkin function.... I thought I had but I may have gotten distracted.

Reply:
Same - not sure.